ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -73,18 +73,18 @@ private[platform] object CompletionFromTransaction {
           Some(commandId),
           transactionId,
           Some(`appId`),
-          Some(submitter),
+          actAs,
           _,
           _,
           recordTime,
           _,
-          _)) if parties(submitter) =>
+          _)) if actAs.forall(parties) =>
```

Comment:
Should this not rather be `exists`?  If any of the acting parties of a transaction is contained in the parties requesting the completions, it should be returned.